### PR TITLE
Fix : Updated Scale Decouple Optimization (for Variance)

### PR DESCRIPTION
**Pull request by dluuo**
Reverted Variance Scale Decouple Optimization for GMM class, and Distribution Loss class StudentT/Normal.

Current Scale Decouple:
```
eps = (eps * scale)
stds = F.softplus(stds) + eps
```

In this PR:
```
stds = F.softplus(stds)
stds = (stds + eps) * scale
```

We empirically tested the scale_decouple versions on hierarchical `Traffic, Tourism, TourismLarge, Wiki, Labour` datasets, and found that with carefully tuned learning rates the second version improves on the first.

### CODE REVIEW

File: nbs/losses.pytorch.ipynb [hunk 1]
```diff
@@ -1127,12 +1127,13 @@
     "    variance and residual location based on anchoring `loc`, `scale`.\n",
     "    Also adds StudentT domain protection to the distribution parameters.\n",
     "    \"\"\"\n",
+    "    print(\"new student scale decoouple\")\n",
     "    df, mean, tscale = output\n",
+    "    tscale = F.softplus(tscale)\n",
```
Comment (kdgutier) [inline nbs/losses.pytorch.ipynb:1132]: Can we take this print out?

**Review by kdgutier — APPROVED** [review]

File: nbs/losses.pytorch.ipynb (hunk 1 above)
Comment (dluuo) [reply nbs/losses.pytorch.ipynb:1131]: resolved

**Review by kdgutier — APPROVED** [review]
